src/devices/src/legacy/mod.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+mod serial;
+pub use serial::Error as SerialError;
+pub use serial::SerialWrapper;
+
+#[cfg(target_arch = "aarch64")]
+mod rtc;
+#[cfg(target_arch = "aarch64")]
+pub use rtc::RtcWrapper;
```

Comment:
I just noticed this, but can you please define modules first, and then have the `use this`?

```rust
mod serial;
#[cfg(target_arch = "aarch64")]
mod rtc;

#[cfg(target_arch = "aarch64")]
pub use rtc::RtcWrapper;
pub use serial::Error as SerialError;
pub use serial::SerialWrapper;

....
```

This makes it easier to see the modules and what we are exporting.